numpy/core/multiarray.py

```diff
@@ -251,6 +253,44 @@ def concatenate(arrays, axis=None, out=None, *, dtype=None, casting=None):
     return arrays
 
 
+def concat(arrays, axis=None, out=None, dtype=None, casting=None):
+    """
+    concat((a1, a2, ...), axis=0, out=None, dtype=None, casting="same_kind")
+
+    Join a sequence of arrays along an existing axis.
+    
+    Parameters
+    ----------
+    a1, a2, ... : sequence of array_like
+        The arrays must have the same shape, except in the dimension
+        corresponding to `axis` (the first, by default).
+    axis : int, optional
+        The axis along which the arrays will be joined.  If axis is None,
+        arrays are flattened before use.  Default is 0.
+    out : ndarray, optional
+        If provided, the destination to place the result. The shape must be
+        correct, matching that of what concatenate would have returned if no
+        out argument were specified.
+    dtype : str or dtype
+        If provided, the destination array will have this dtype. Cannot be
+        provided together with `out`.
+        .. versionadded:: 1.20.0
+    casting : {'no', 'equiv', 'safe', 'same_kind', 'unsafe'}, optional
+        Controls what kind of data casting may occur. Defaults to 'same_kind'.
+        .. versionadded:: 1.20.0
+
+    Returns
+    -------
+    res : ndarray
+        The concatenated array.
+
+    Notes
+    -----
+    concat is an alias of concatenate
```

Comment:
Turn this into a `See Also` section, carefully copying the formatting to get the links to work correctly. In the `Notes`, mention that there is full documentation in the concatenate documentation.